src/Oci8/Query/Grammars/OracleGrammar.php

```diff
@@ -160,11 +161,11 @@ public function wrapTable($table)
             $table = str_replace(' as ', ' ', $table);
         }
 
-        return $this->getSchemaPrefix() . $this->wrap($this->tablePrefix . $table, true);
+        return $this->getSchemaPrefix() . $this->tablePrefix . $table;
```

Comment:
Table names are allowed to be wrapped in double quotes. See: https://docs.oracle.com/cd/B28359_01/server.111/b28286/sql_elements008.htm